src/components/application_manager/include/application_manager/commands/command_impl.h

```diff
@@ -201,7 +201,7 @@ class CommandImpl : public Command {
   CommandParametersPermissions removed_parameters_permissions_;
 
 #ifdef ENABLE_LOG
-  static log4cxx::LoggerPtr logger_;
+  static const log4cxx::LoggerWeakPtr logger_;
```

Comment:
@SNiukalov Can you please explain why did you make it static const? I mean what is the sense here: to create a static pointer to unchangeable value?